dependencies.gradle

```diff
@@ -3,24 +3,35 @@ ext.versions = [
         name                : '0.1',
 
         minSdk              : 19,
-        targetSdk           : 24,
+        targetSdk           : 22,
```

Comment:
Or in Config hard (constants = BuildConfig::class, manifest = "AndroidManifest.xml", sdk = intArrayOf(Build.VERSION_CODES.LOLLIPOP))